### DISCUSSION TITLE
added my own extension to Expecto

### DISCUSSION THREAD

**Discussion by tonyx** [haf/expecto#513] [Ideas]
Hi, my name is Tony Lucca. I included few extensions to Expecto in a lib I am working on (https://github.com/tonyx/Sharpino). Basically I added functions to parameterized tests like multipleTestCase/fmultipleTestCase/pmultipleTestCase that works as follows:         multipleTestCase "should do A, B, C" appVersionsEnvs <| fun (setup, _, service) -> ... where the appVersionsEnvs is a list of triplets containing specific setup and service functions (to obtain a sort of parametrized tests). Not sure if  something similar was already in Expecto but in case you are interested in including my proposal I can submit a proper P.R. if you ask me to do it. Thanks. bye.

**Comment by farlee2121** [comment]
I made a similar library https://github.com/farlee2121/Expecto.TestApi

I never contributed it because [TestFixture](https://github.com/haf/expecto?tab=readme-ov-file#test-fixtures) already accomplishes the same goal. 

Thanks for the call out though
